# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/fs/parent/datamodel/FileSystemDataManager.cpp
# Commit: 399dce373c59
# Full Hash: 399dce373c5917ac141f7fd341ed7c4dca56144a
# Author: Jan Varga <jvarga@mozilla.com>
# Date: 2022-08-31 21:54:47
# Regressor Bug: 1786465
# File Overlap Count: 3
# Description:
#   Bug 1786465 - Move FileSystemDataManager to own source files; r=dom-storage-reviewers,jesup
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D155550
# ==============================================================================

diff -r f62bd47cabd3 -r 399dce373c59 dom/fs/parent/datamodel/FileSystemDataManager.cpp
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/dom/fs/parent/datamodel/FileSystemDataManager.cpp	Wed Aug 31 09:34:54 2022 +0000
@@ -0,0 +1,19 @@
+/* -*- Mode: C++; tab-width: 8; indent-tabs-mode: nil; c-basic-offset: 2 -*- */
+/* vim: set ts=8 sts=2 et sw=2 tw=80: */
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "FileSystemDataManager.h"
+
+#include "mozilla/Result.h"
+
+namespace mozilla::dom::fs::data {
+
+FileSystemDataManager::result_t
+FileSystemDataManager::CreateFileSystemDataManager(
+    const fs::Origin& /*aOrigin*/) {
+  return nullptr;
+}
+
+}  // namespace mozilla::dom::fs::data